### ISSUE TITLE
Crash: Cannot move between individual entities in the scanner list

### ISSUE THREAD

**Issue by ahicks92** [Factorio-Access/FactorioAccess#147]
The traceback:

```
b'  27.927 Error MainLoop.cpp:1404: Exception at tick 1189644: The mod Factorio Access Mod (0.11.2) caused a non-recoverable error.\r\n'
b'Please report this error to the mod author.\r\n'
b'\r\n'
b'Error while running event FactorioAccess::scan-selection-down (ID 366)\r\n'
b"__FactorioAccess__/control.lua:4446: attempt to index field '?' (a nil value)\r\n"
b'stack traceback:\r\n'
b'\t__FactorioAccess__/control.lua:4446: in function <__FactorioAccess__/control.lua:4411>\r\n'
```

The save is attached.

This is probably high pri because whatever I have done is 100%.  Switch to the enemy category, find the biters, then shift+pgup/pgdown.  Both seem to do it no problem.  Not sure why, going to pass directly to @LevFendi 
'
Marked it bug light because I'm pretty sure that this is a missing nil check that's obvious to anyone who has a grasp on the scanner code.

[main.zip](https://github.com/Factorio-Access/FactorioAccess/files/15397406/main.zip)

**Comment by LevFendi** [comment]
The issue was that this part of the scanner code has category checks that were not updated during the review. I have fixed it and cleaned it up in commit fea33df14f809ca06d73f5a7e8cb0df2a0a3ce5f . I will PR this now.

**Comment by LevFendi** [comment]
Fixed and tested and merged.
